server/events/apply_command_runner.go

```diff
@@ -31,6 +32,7 @@ func NewApplyCommandRunner(
 		dbUpdater:           dbUpdater,
 		DB:                  db,
 		parallelPoolSize:    parallelPoolSize,
+		SilenceNoProjects:   SilenceNoProjects,
```

Comment:
Any reason for this to be capitalized?

Reply:
@msarvar I had to export the variable to be able to override it in the tests. un-capitalized vars are not exported in Go.